practice/case.py

```
@@ -0,0 +1,218 @@
+""" CASE CONVERSION """
+
+
+def prompt_variable():
+    var_name = input('Variable name (snake_case): ')
+    if var_name == '':
+        var_name = 'hello_world'
+
+    return var_name
+
+
+def normalize_case(var_name, from_style):
+    """
+    >>> normalize_case('branch_test', 'snake')
+    ['branch', 'test']
+    >>> normalize_case('branch_test', 'camel')
+    ['Branch_test']
+    """
+
+    if from_style == 'snake':
+        normalized = normalize_snake(var_name)
+    elif from_style == 'camel':
+        normalized = normalize_camel(var_name)
+    elif from_style == 'kebab':
+        normalized = normalize_kebab(var_name)
+    elif from_style == 'constant':
+        normalized = normalize_constant(var_name)
+    elif from_style == 'space':
+        normalized = normalize_space(var_name)
+    else:
```

I think this is another instance in which you should throw an exception. The code does not know how to handle this case, so no need to even try!